cmd/dt/pull.go

```diff
@@ -63,6 +64,8 @@ func newPullCommand() *cobra.Command {
 			if imagesDir == "" {
 				imagesDir = chart.ImagesDir()
 			}
+
```

Comment:
I think it would be easier to just check the lock and show the warning if no images are found instead of actually trying to pull them:

```
// get the lock
lock := ...
if len(lock.Images) == 0 {
     l.Warnf("No images found in Images.lock")
    return nil
}

```

The only drawback is getting the lock here, and if we have images, getting it again in pullChartImages. But we could add a new method to chartutils.Chart to get the Images.lock, caching it the first time is called.

In any case, even if we call twice, I think it simplifies the code and feels more natural to just let the user know there are no images right away than to start logging that we are going to pull them and then say we have nothing to pull.